bootstrap-demo/kubernetes-dns/src/main/scala/akka/cluster/bootstrap/KubernetesHealthChecks.scala

```diff
@@ -18,7 +18,16 @@ class KubernetesHealthChecks(system: ActorSystem) {
   //#health
   private val readyStates: Set[MemberStatus] = Set(MemberStatus.Up, MemberStatus.WeaklyUp)
   private val aliveStates: Set[MemberStatus] =
-    Set(MemberStatus.Joining, MemberStatus.WeaklyUp, MemberStatus.Up, MemberStatus.Leaving, MemberStatus.Exiting)
+    Set(
+      // 'Removed' might be counter-intuitive here, but the cluster status is also 'removed' during pre-join
+      // initialization (https://github.com/akka/akka/issues/25663)
+      MemberStatus.Removed,
```

Comment:
This results in the counter-intuitive consequence that a "Down" node is considered dead, but after it has transitioned to "Removed" it is considered alive.

Reply:
I commented on the ticket 
https://github.com/akka/akka-management/issues/312#issuecomment-423885488